### PR TITLE
Adding level-zero package

### PR DESCRIPTION
**Pull request by kurapov-peter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kurapov-peter** [comment]
@oleksandr-pavlyk, could you please confirm here you are OK with being on the maintainer list?

**Comment by oleksandr-pavlyk** [comment]
Yes, I am willing to be on the maintainer list for the level-zero loader feedstock.

File: recipes/level-zero/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/bash
+
+mkdir build
+mkdir install
+
+cmake -DCMAKE_INSTALL_PREFIX="${SRC_DIR}/install" \
+    -S ${SRC_DIR}/level-zero \
+    -B ${SRC_DIR}/build \
+    -Wno-dev
+cmake --build ${SRC_DIR}/build --config Release
```
Comment (isuruf) [inline recipes/level-zero/build.sh:10]:
```suggestion
cmake --build ${SRC_DIR}/build --config Release -- -j${CPU_COUNT}
```

File: recipes/level-zero/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (isuruf) [inline recipes/level-zero/LICENSE:1]: This file is not needed. conda-build can find the one from the source tarball.

Comment (kurapov-peter) [reply recipes/level-zero/LICENSE]: Thanks, removed

File: recipes/level-zero/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+         - test -f ${PREFIX}/lib/pkgconfig/level-zero.pc  # [linux]
+
+about:
+  home: https://github.com/oneapi-src/level-zero
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Intel(R) level zero libraries'
+
+extra:
```
Comment (isuruf) [inline recipes/level-zero/meta.yaml:79]:
```suggestion
extra:
  feedstock-name: level-zero
```

File: recipes/level-zero/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
+{% set version = "1.9.9" %}
+{% set test_version = "1.9.0" %}
```
Comment (oleksandr-pavlyk) [inline recipes/level-zero/meta.yaml:2]: @kurapov-peter If the `test_version` is always guaranteed to be the first two digits of `version` and a zero, perhaps we can be uses Jinja here:

```
{% set test_version = ".".join(version.split('.')[:2] + ['0']) %}
```

This way `'test_version'` will always stay in sync with the `'version'`.

Comment (kurapov-peter) [reply recipes/level-zero/meta.yaml]: Sure, updated the meta.yaml

**Comment by kurapov-peter** [comment]
Anything else folks? @isuruf, @oleksandr-pavlyk

**Pull request merged**